src/generators/operationGenerator.ts

```diff
@@ -106,11 +107,16 @@ function buildSpec(spec: OperationSpecDetails): string {
   const queryParams = buildParameters(spec, "queryParameters");
   const urlParams = buildParameters(spec, "urlParameters");
   const headerParams = buildParameters(spec, "headerParameters");
+  const isXML = spec.isXML ? "isXML: true," : "";
+  const contentType =
+    spec.requestBody && isXML
+      ? "contentType: 'application/xml; charset=utf-8',"
+      : "";
   return `{ path: "${spec.path}", httpMethod: "${
     spec.httpMethod
   }", responses: {${responses.join(
     ", "
-  )}},${requestBody}${queryParams}${urlParams}${headerParams}serializer
+  )}},${requestBody}${queryParams}${urlParams}${headerParams}${isXML}${contentType}serializer
```

Comment:
Need to cleanup this, probably a better idea to use ts-morph writers instead of building the strings this way. Will explore this in a separate PR filed #560

Reply:
It's not the worst way to do it since Prettier will break up the lines correctly anyway.  There's probably better ways to spend your time for now :)